stat.h

```diff
@@ -237,6 +237,12 @@ struct thread_stat {
 	fio_fp64_t ss_deviation;
 	fio_fp64_t ss_criterion;
 
+	bool priority_bit;
```

Comment:
bool probably isn't a good choice for an on-wire variable type. Should be uint8_t or something like that. This structure is sent over the wire.